### ISSUE TITLE
Build status on README.md shows failing.

### ISSUE THREAD

**Issue by kdbanman** [jakobehn/GitFlow.VS#24]
Specifically, the API call

    https://jakob.visualstudio.com/DefaultCollection/_apis/public/build/definitions/da33bffd-4752-4c26-962d-a83d85e71d2f/114/badge

shows this image

![badge](https://cloud.githubusercontent.com/assets/1323521/9882713/5a91f5ac-5b93-11e5-869b-cbf865f170d8.png)

**Comment by kdbanman** [comment]
I tried to peek at the build definition and the logs, but it seems the [build](https://jakob.visualstudio.com/DefaultCollection/_apis/public/build/definitions/da33bffd-4752-4c26-962d-a83d85e71d2f/114) (or the whole project?) isn't publicly accessible.

**Comment by jakobehn** [comment]
Yes, that is part of a build definition at Visual Studio Team Services. Unfortunately VSTS currently doesn't allow anonymouse access (other than the badge information). 

Since the project have some specific SDK requirements it's not easy to make it compile correctly at for example AppVeyor. But I might give it a go again
